extension/src/paymentHandler/payment-utils.js

```diff
@@ -59,25 +59,25 @@ function getMatchingCtpState (adyenState) {
 
 function createAddInterfaceInteractionAction (
   {
-    request, response, type, status
+    request, response, type
   }
 ) {
   // strip away sensitive data
   delete response.additionalData
 
   return {
     action: 'addInterfaceInteraction',
-    type: { key: c.CTP_INTERFACE_INTERACTION },
+    type: { key: c.CTP_INTERFACE_INTERACTION_PAYMENT_TYPE_KEY },
     fields: {
-      createdAt: new Date(),
+      createdAt: new Date().toISOString(),
```

Comment:
> I was also doing it, but sometimes I get a weird output, so I changed it

Thanks for that suggestion, I am trying to understand that argument, which looks like your experience with toISOString was not good, I guess your problem could be related to Date() object.. Any idea how we could solve it or any alternative also could be a solution.

##### Why I added it ?

A typical createdAt if I use toISOString() will look sth like: "createdAt": "2020-04-07T07:44:22.132Z"
the values for the payment resource will look like: 
"createdAt": "2020-04-07T07:44:22.249Z",
"lastModifiedAt": "2020-04-07T07:44:22.249Z",

without using iso string, it will look like: "createdAt": "2020-04-06T09:17:19.829Z",
as you would know it will use my local date instead of UTC.

Reply:
If you tested it and it looks ok, then no problem.

Reply:
related issue: https://github.com/commercetools/commercetools-adyen-integration/issues/75#issuecomment-514196727